Client/Application/AppDelegate.swift

```diff
@@ -31,7 +31,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         log.info("startApplication begin")
 
         self.window = UIWindow(frame: UIScreen.main.bounds)
-
+        window?.tintColor = .theme.ecosia.primaryBrand
```

Comment:
@ecotopian @d4r1091 @lucaschifino I've set the global tint color so e.g. the file picking dialog also picks it up, but not sure if there might be any unintended side-effect on other areas. Just adding this comment to raise awareness.

Reply:
I can't give a guarantee to exclude side effects, but I think that the strategy is generally okay to tint system like dialogues. Still I think you need to add the functionality that it reassigns the color on theme changes.

Reply:
Done ✅